src/pages/Admin/Room/Edit/RoomEditModal.types.ts

```diff
@@ -1,6 +1,6 @@
 import { RoomData_v2 } from "types/rooms";
 
-export interface EditRoomProps {
+export interface RoomEditModalProps {
   isVisible: boolean;
```

Comment:
Can we please move this type definition into the `src/pages/Admin/Room/Edit/RoomEditModal.tsx` file (and remove this file entirely after that), placed just above the component definition like our standard patterns. This separate `foo.types.ts` is a pattern that we want to remove from our codebase.